tests/testcases/examples/users/pk/get.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Information about a user",
+    "url": "/users/<pk>",
```

Comment:
At some point, I added a `notes` field here for extra documentation that could be useful.